scripts.py

```diff
@@ -194,7 +194,7 @@ def __init__(self, _db=None, cmd_args=None, manager=None, *args, **kwargs):
         super().__init__(_db, *args, **kwargs)
         self.parse_args(cmd_args)
         if not manager:
-            manager = CirculationManager(self._db)
+            manager = CirculationManager(self._db, MagicMock())
```

Comment:
I'm uncomfortable about having MagicMock in the app code. Why do we need it here? Are there reasonable alternatives?

Reply:
Good catch. I'm not sure what I was thinking here. I must have thought this was a test mock class. I updated it to use the scripts container property.